util/panic-hook/Cargo.toml

```diff
@@ -7,4 +7,4 @@ version = "0.1.0"
 authors = ["Parity Technologies <admin@parity.io>"]
 
 [dependencies]
-backtrace = "0.3.2"
+backtrace = "0.3.43"
```

Comment:
I had weird problems with the lockfile and the `cc` crate: `rust-secp256k1` has a hard requirement on `cc@1.0.41` which caused cascading trouble that is visible here. I can try to revert this change if anyone feels it's risky/irrelevant.